### PR TITLE
test03-bacic_tests

### PR DESCRIPTION
**Pull request by dianaeldemerdash**

### CODE REVIEW

File: .idea/1lims-automation.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (0xIslamTaha) [inline .idea/1lims-automation.iml]: please, don't push `.idea` folder nor the `.pyc`

File: ui_testing/readme.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ cd ui_testing
 # run in head mode
 nosetests-3.4 -vs --logging-level=WARNING --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
 # run in headless mode
-xvfb-run -a nosetests-3.4 -vs --logging-level=WARNING  --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
+-run -a nosetests-3.4 -vsxvfb --logging-level=WARNING  --with-flaky --force-flaky --max-runs=3 --no-flaky-report testcases/basic_tests --tc-file=../config.ini --tc=site.password:admin
```
Comment (0xIslamTaha) [inline ui_testing/readme.md]: this is wrong, revert it

File: ui_testing/testcases/basic_tests/test03_orders.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+         New: Orders: Cancel Button Approach: In case I update the material type field then press on cancel button
+         message will appear ( data will be lost )
+         LIMS-4281
+         :return:
+         """
+         self.order_page.get_random_order()
+         order_url = self.base_selenium.get_url()
+         self.order_page.sleep_tiny()
+         current_material_type = self.order_page.get_material_type()
+         self.order_page.set_material_type(random=True)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test03_orders.py]: I'm wondering if you could run this test successfully cause this method doesn't have `random` as a parameter in any way

**Comment by 0xIslamTaha** [comment]
@dianaeldemerdash please, check the comments and fix it

File: ui_testing/readme.md (hunk 2 above)
Comment (dianaeldemerdash) [reply ui_testing/readme.md]: @0xIslamTaha  Please clarify this comment , Do you mean that I run with nosetests3 and this readme.md file execute from nosetests-3.4

**Pull request closed without merging**